frontend/packages/client/src/pages/index.js

```diff
@@ -25,6 +27,8 @@ export default function AppPages() {
               <Home />
             </Route>
             <Route exact path="/about" component={About} />
+            <Route exact path="/privacy-policy" component={PrivacyPolicy} />
+            <Route exact path="/terms-of-service" component={TermsOfService} />
```

Comment:
Do you know if we are adding a button or a nav link to get to these routes? besides the pages that have links